setup.py

```diff
@@ -66,7 +66,7 @@
     ],
     long_description=long_description,
     long_description_content_type="text/markdown",
-    python_requires=">=3.8",
+    python_requires=">=3.7",
```

Comment:
if nvflare doesn't work in python 3.9+, we should add a python_requires<3.9 also

Reply:
that I don't know though, should we add that to be safe?

Reply:
@IsaacYangSLA @yhwen might know this

Reply:
I think we could verify (and possibly fix) this in another PR what do you think?

Reply:
Makes sense, this PR is just about 3.7

Reply:
I think since we have really tried nvflare in 3.9 or 3.10, we should use
python_requires=">=3.7,<3.9"